modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,76 @@
+= Anypoint Monitoring and Visualizer
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Requirements
+
+=== Stateful Set Directories
+
+Each stateful set uses a different directory to hold its data.
+
+* InfluxDB meta pods (3 pods): `/var/lib/data/influxdb/meta`
+* Data pods (2 pods): `/var/lib/data/influxdb/data`
+* Logstash: `/var/lib/data/logstash`
+* Insight: `/var/lib/data/dias-cassandra` and `/var/lib/data/dias-meld`
+
+=== Port Requirements
+
+The required ports must be accessible in the main balancer, including port 8895.
+
+== Backup and Restore
```

Comment:
This needs to be merged with the PCE backup-restore?

Reply:
Good point. I moved the backup and restore instructions to the page "Configure Backup and Restore for Anypoint Private Cloud".